src/styles/_layout.scss

```diff
@@ -6,9 +6,9 @@ body,
 .pf-c-page__main-section {
   /*
    * --pf-c-page__header--MinHeight is the height of the masthead
-   * 82px is the vertical space being used by the breadcrumb and tab headers
+   * 130px is the vertical space being used by the breadcrumb, tab headers, and the name of the {workload,svc,app}
    */
-  --kiali-details-pages-tab-content-height: calc(100vh - 82px - var(--pf-c-page__header--MinHeight));
+  --kiali-details-pages-tab-content-height: calc(100vh - 130px - var(--pf-c-page__header--MinHeight));
```

Comment:
Yeah, I wish we could get away from all of the pixel calculations.

Reply:
I think that the right way to make this happen is to contribute to patternfly-react. Anything we do in kiali-ui will involve hacking the PF css.